apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -284,6 +310,9 @@ export default class ExportCKLModal extends Vue {
   @Prop({type: Object, required: true}) readonly filter!: Filter;
 
   showingModal = false;
+  formatProfileTitle = false;
+  updatingProfileTitle = false;
```

Comment:
I'm not sure what this does. I see it gets set to true when the name needs to be reformatted, but it doesn't get used at any point. Does this get used somewhere outside of these files?

Reply:
The flag updatingProfileTitle is no longer needed, it was used on a previous logic, will remove.